src/Button/Button.less

```diff
@@ -21,6 +21,7 @@ a, button {
       color: #4B4B59;
       background-color: #D7D9D9;
       border: none;
+      border: 1px solid #D7D9D9;
```

Comment:
not tied to this specific solution. we can use hard heights + border-box sizing for the buttons instead, if anyone feels strongly